src/util/host.js

```diff
@@ -58,3 +57,22 @@ export function getBuildPath () {
 
   return buildPath;
 }
+
+function getLocalDappsPath () {
+  // FIXME
+  const userData = typeof window !== 'undefined'
+    ? window.require('electron').remote.app.getPath('userData')
+    : require('electron').app.getPath('userData');
+
+  return path.join(userData, 'dapps');
+}
+
+// FIXME
+// Need to find a better way to be able to import this file
+// from electron/ as well as src/
```

Comment:
I think it's cleaner to have 2 separate util/host.js, one for browser and one for electron, maybe even a better name than host for the electron one.